langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -106,6 +126,11 @@ public isolated function ensureType(any|error v, typedesc<any> t = <>) returns t
 #
 # Returns true if read-only, false otherwise.
 #
+# ```ballerina
+# int[] & readonly finalScores = [21, 12, 33, 45, 81];
```

Comment:
```suggestion
# int[] scores = <readonly> [21, 12, 33, 45, 81];
```

Wondering if we should do something like this. since with the current version, the `isReadOnly` check is kinda redundant.